src/SFML/Window/OSX/HIDInputManager.hpp

```diff
@@ -103,21 +93,64 @@ class HIDInputManager : NonCopyable
     ///
     /// Return sf::Keyboard::Unknown if it doesn't match any 'localized' keys.
     ///
-    /// By 'localized' I mean keys that depend on the keyboard layout
-    /// and might not be the same as the US keycode in some country
-    /// (e.g. the keys 'Y' and 'Z' are switched on QWERTZ keyboard and
+    /// By 'localized' we mean keys that depend on the keyboard layout
+    /// and might not be the same as the US keycode for some countries
```

Comment:
`s/countries/keyboard layouts`

Reply:
This text is part of the USB HID documentation. I believe it is correct from an historical point of view, at least.